src/modules/previewpane/SvgPreviewHandler/Utilities/SvgPreviewHandlerHelper.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace SvgPreviewHandler.Utilities
+{
+    /// <summary>
+    /// Helper utilities for Svg Preview Handler.
+    /// </summary>
+    public class SvgPreviewHandlerHelper
+    {
+        /// <summary>
+        /// Dictionary of elements that are blocked from Svg for preview pane.
+        /// </summary>
+        private static Dictionary<string, bool> blockedElementsName = new Dictionary<string, bool>
+        {
+            { "script", true },

```

Comment:
Script tags should always be blocked. But images are conditionally blocked. So should we display message for that?

Reply:
Currently i'm blocking all the image tags. Not sure if we should allow relative path images as well.